cogs/subscriptions.py

```diff
@@ -244,6 +252,98 @@ async def on_message(self, ctx):
                 except discord.Forbidden:
                     pass
 
+    @commands.Cog.listener()
+    async def on_message_edit(self, before, after):
+        # Obtains the relebent data.
+        subs = data_handler.load("subscriptions")
+
+        # Stops a chain response.
+        if self.bot.user.id == before.author.id:
+            return
+
+        # Creates a list of ids of valid subscriptions
+        channels = list(subs["Subscriptions"].values())
+        # Check if the channel is in that list of valid subscriptions we just made
+        if before.channel.id in channels:
+
+            # Cycle through the subscibed channels of the subscription
+            for channelID in subs["Subscribers"][str(before.channel.id)]["Channels"]:
+
+                # Get the TextChannel object from the id
+                channel = self.bot.get_channel(channelID)
+
+                dayAgo = datetime.datetime.now() - datetime.timedelta(days=1)
+
+                msg = None
+
+                async for mesg in channel.history(limit = 200, after = dayAgo):
+                    if before.content in mesg.content and mesg.author.id == self.bot.user.id:
+                        msg = mesg
+                # Convert the message so we can output it all correctly.
+                # Checks if it's a system message, if so, output it nicely.
+                if after.is_system():
+                    message = f">>> {after.system_content}"
+                # Checks if the content is empty, if no, just add the author's name.
+                elif after.content == "":
```

Comment:
" " <--- wouldn't this be empty, too?

Reply:
Empty is "" I think You can't have a message with a content of " ". `None` could also work. I'll have to check this on the d.py server.

Reply:
Using "" is correct, as that's an empty string.